src/jit/gentree.cpp

```diff
@@ -1791,6 +1791,11 @@ regMaskTP GenTreeCall::GetOtherRegMask() const
     return resultMask;
 }
 
+bool GenTreeCall::IsPure(Compiler* compiler) const
+{
```

Comment:
> Someday we could determine if a Jitted method being called is Pure

That's why I named the method `IsPure`: if we ever gain the ability to decide whether or not an arbitrary method is pure, we can add that logic to this method.